LayoutTestBase/UIViewHelpers/UIView+LYTFrameComparison.m

```diff
@@ -27,10 +27,35 @@ - (BOOL)lyt_before:(UIView *)otherView {
     }
 }
```

Comment:
You should convert the versions that don't have the `fromCenter` parameter to call the `fromCenter` version with `NO` as the param. That way, we don't have duplicated code.

Reply:
Done, there's only one old implementation body, lyt_before. I changed it to call the new code